README.md

```diff
@@ -112,6 +112,15 @@ In Powershell:
 
 Note that OpenSSL isn't needed on Windows.
 
+### Setup on OpenBSD
```

Comment:
Hey @jasperla, could you update these docs with the version so they're more accurate?